src/App.tsx

```diff
@@ -89,10 +89,22 @@ const Settings = React.lazy(() =>
 import { withGlobalContext } from 'hooks/useGlobalState'
 import { rootReducer, INITIAL_STATE } from 'reducers-actions'
 import PrivateRoute from './PrivateRoute'
+import { assertNonNull } from 'utils'
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const Router: typeof BrowserRouter & typeof HashRouter = (window as any).IS_IPFS ? HashRouter : BrowserRouter
 
+function getInitialUrl(): string {
+  // Validate tha
+  assertNonNull(CONFIG.initialTokenSelection, 'initialTokenSelection config is required')
+  const { sellToken: initialSellToken, receiveToken: initialReceiveToken } = CONFIG.initialTokenSelection
+  assertNonNull(initialSellToken, 'sellToken is required in the initialTokenSelection config')
+  assertNonNull(initialReceiveToken, 'receiveToken is required in the initialTokenSelection config')
+  return '/trade/' + initialSellToken + '-' + initialReceiveToken + '?sell=0&price=0'
```

Comment:
I remember us having problems with some token symbols with special chars and dashes `-`
Likely need to do same here,who knows what default a fork can choose

`import { encodeTokenSymbol, decodeSymbol } from '@gnosis.pm/dex-js'` etc

Reply:
hehe, ok. I'm happy if we just support normal tokens. anyways, I'll add that, thx!

Reply:
That function requires an address as mandatory param, which is not loaded at this point
I think in the super edge case of someone wanting a token with dashes in their name, they can encode it themselves.

Reply:
`import { encodeSymbol } from '@gnosis.pm/dex-js'`